scipy/signal/_upfirdn.py

```diff
@@ -105,6 +117,21 @@ def upfirdn(h, x, up=1, down=1, axis=-1):
         The axis of the input data array along which to apply the
         linear filter. The filter is applied to each subarray along
         this axis. Default is -1.
+    mode : str, optional
+        The signal extension mode to use. The set
+        ``{"constant", "symmetric", "reflect", "edge", "wrap"}`` correspond to
+        modes provided by ``numpy.pad``. ``"smooth"`` implements a smooth
```

Comment:
```suggestion
        modes provided by :func:`numpy.pad`. ``"smooth"`` implements a smooth
```

Reply:
Is it even necessary to have :func:? I think SciPy has set `autolink` in `conf.py` so maybe just single backticks should work. I will test it locally and see

Reply:
Just the single backticks should work, yeah